lib/bot.js

```diff
@@ -36,6 +36,11 @@ function launchBot (team, cb) {
   if (!team.bot) { return cb(new Error('team has no bot: ' + team)) }
   const bot = slack.rtm.client()
   const token = team.bot.bot_access_token
+  bot.team_join( data => {
+    require('./events/team_join')(token, data, e => {
+      console.log('sent welcome message')
+    })
+  })
```

Comment:
I think I'd prefer this even be tracked through the [Events API](https://api.slack.com/events-api#begin) -- I've enabled the `team_join` event on both slack apps, so you should be able to add an entry to `lib/events`